### DISCUSSION TITLE
Add theme persister to the docs site

### DISCUSSION THREAD

**Discussion by artalar** [mantinedev/mantine#4865] [Feature requests]
Hi! My team uses Mantine with a custom theme, and I thought it would be cool to reuse the documentation site as a component gallery instead of building our own Storybook from scratch. Could you add an input for a JSON theme object that will persist the theme to `localStorage` and use the persisted theme in all example pages?

Thank you!

**Comment by rtivital** [comment]
Mantine theme is static. If you need this feature, you can download the source code and change the theme there and deploy the website with your theme.
